### PR TITLE
add status_timestamp and authorization_expires_at

### PR DESCRIPTION
**Pull request by dougperez**
Adds `status_timestamp` and `authorization_expires_at`

### CODE REVIEW

File: lib/conduit/braintree/json/transaction.rb [hunk 1]
```diff
@@ -19,13 +19,23 @@ def transaction_attributes
...
           else
             value = response.respond_to?(attr_name) ? response.send(attr_name) : nil
             h.merge(attr_name.to_s => value)
           end
         end
       end
+
+      def get_updated_status(transaction)
+        status_history = Array(transaction.status_history).compact
+        status_history.last&.timestamp
```
Comment (pedromartinez) [inline lib/conduit/braintree/json/transaction.rb]: Is this reliable enough to infer that the current status and the last history item are showing the same status?

Comment (dougperez) [reply lib/conduit/braintree/json/transaction.rb]: Hm, I guess it could be wrong. I could select the message where the status matches.

Comment (dougperez) [reply lib/conduit/braintree/json/transaction.rb]: Updated

File: lib/conduit/braintree/json/transaction.rb [hunk 2]
```diff
@@ -19,13 +19,23 @@ def transaction_attributes
...
           else
             value = response.respond_to?(attr_name) ? response.send(attr_name) : nil
             h.merge(attr_name.to_s => value)
           end
         end
       end
+
+      def get_status_timestamp(transaction)
+        status_history = Array(transaction.status_history).compact
+        status_history.find { |sh| sh.status == transaction.status  }&.timestamp
```
Comment (pedromartinez) [inline lib/conduit/braintree/json/transaction.rb]: Double space after status

**Pull request merged**